src/Bicep.LangServer/Telemetry/TelemetryEvent.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+
+namespace Bicep.LanguageServer.Telemetry
+{
+    public class TelemetryEvent : TelemetryEventParams
```

Comment:
What about naming this `BicepTelemetryEvent` just to distinguish it from the Omnisharp-provided library?

Reply:
Updated here:
https://github.com/Azure/bicep/commit/ffbef4cf981f55f6966f50de6ed4dec09bd90905